qtkeychain/keychain.cpp

```diff
@@ -35,6 +35,11 @@ void Job::setSettings(QSettings *settings)
 
 void Job::start()
 {
+    if (d->started) {
```

Comment:
Do we have to take care about multithreaded use? Sorry, I don't know about the general policy on whether Qt Keychain supports multithreaded access or not.